docker/alpine/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.12-alpine3.10 as builder
+
+RUN apk add --no-cache git gcc g++ ca-certificates \
+  && go get -u github.com/drakkan/sftpgo
+
+FROM alpine:3.10
+
+RUN  apk add --no-cache ca-certificates su-exec \
+  && mkdir -p /data /etc/sftpgo
+
```

Comment:
copy a default config inside the provider and allow the user to override it. sftpgo will search config files in the following folders:

1) config dir
2) $HOME/.config/sftpgo
3) /etc/sftpgo
4) current directory

if no config file is found, in your case if the user put no config inside the exposed volume, env var override does not work

Reply:
see my comment below

Reply:
Ohh .. I just got it, correct me if I'm wrong. Is it necessary to configure a configuration file for the environment variables to work?

Reply:
Yes, without a config file env vars will not work